sw/host/cryptotest/testvectors/data/BUILD

```diff
@@ -131,3 +131,31 @@ run_binary(
         ("ecdh_secp384r1_test.json", "wycheproof_ecdh_p384"),
     ]
 ]
+
+[
+    run_binary(
+        name = cryptotest_name,
+        srcs = [
+            "@sphincsplus_kat//:{}".format(src_name),
+            "//sw/host/cryptotest/testvectors/data/schemas:sphincsplus_schema.json",
+        ],
+        outs = [":{}.json".format(cryptotest_name)],
+        args = [
+            "--src",
+            "$(location @sphincsplus_kat//:{})".format(src_name),
+            "--dst",
+            "$(location :{}.json)".format(cryptotest_name),
+            "--schema",
+            "$(location //sw/host/cryptotest/testvectors/data/schemas:sphincsplus_schema.json)",
+            "--hash_alg",
+            hash_alg,
+        ],
+        tool = "//sw/host/cryptotest/testvectors/parsers:rsp_sphincsplus_parser",
+    )
+    for src_name, cryptotest_name, hash_alg in [
+        ("sphincs-sha256-128s-simple/PQCsignKAT_64.rsp", "rsp_sphincsplus_sha256_128s_simple", "sha-256"),
+        ("sphincs-sha256-128s-robust/PQCsignKAT_64.rsp", "rsp_sphincsplus_sha256_128s_robust", "sha-256"),
+        ("sphincs-shake256-128s-simple/PQCsignKAT_64.rsp", "rsp_sphincsplus_shake256_128s_simple", "shake-256"),
+        ("sphincs-shake256-128s-robust/PQCsignKAT_64.rsp", "rsp_sphincsplus_shake256_128s_robust", "shake-256"),
```

Comment:
For what it's worth, we'll probably never need `robust`; NIST already announced that it won't be standardized and it has a ~2x speed penalty. We also currently do not use the `sha256` variant either, so only `shake256-128s-simple`.

Reply:
Removing these make the build rule much simpler, thanks!